network/ipam/concepts.mdx

```diff
@@ -0,0 +1,54 @@
+
+More functionality is planned for the future, including:
+
+- Reserve/release private IP addresses in the console
+- Select a reserved private IP address to use when attaching a resource to a Private Network
+- Management of public IP addresses
+
+## IPv4
+
+Internet Protocol Version 4 is the standard protocol used for IP addresses, and routes most internet traffic as of today. Each IPv4 address has 32 bits. Written in human-readable form, an IPv4 address is generally shown as four octets separated by periods, e.g. `72.16.254.1`.
+
+Scaleway resources may have a [public](#public-flexible-ip-address) IPv4 address (flexible IP) for accessibility from the public internet, and will have a [private](#private-ip-address) IPv4 address for each Private Network they are attached to.
+
+## IPv6
+
+Internet Protocol Version 6 is the most recent version of the IP protocol used for IP addresses. Each IPv6 address has 128 bits. Written in human-readable form, an IPv6 address can be shown as eight groups of four hexadecimal digits, each group representing 16 bits and separated by a colon, e.g. `2001:0DB8:0000:0003:0000:01FF:0000:002E`. This can also be notated as `2001:DB8::3:0:1FF:0:2E`.
+
+IPv6-compatible Scaleway resources may have a [public](#public-flexible-ip-address) IPv6 address (flexible IP) for accessibility from the public internet, and will have a [private](#private-ip-address) IPv6 address for each Private Network they are attached to. 
+
+## Private IP address
+
+Private IP addresses identify devices on Private Networks. They are not routed on the public internet, and can be used to route traffic privately between resources attached to the Private Network, as well as to [other Private Networks in the same VPC](/network/vpc/concepts/#routing).
+
+When a resource joins a Private Network, the Private Network's [DHCP](/network/vpc/concepts/#dhcp) assigns it a private [IPv4](~ipv4) address (and an additional [IPv6](#ipv6) address for IPv6-compatible resources) for identification on that network. This IP is selected from the Private Network's [CIDR](/network/vpc/concepts/#cidr-block) block. The IP address will then not change unless you detach the resource from the Private Network, or delete the resource. 
+
+## Public (flexible) IP address
+
+Public IP addresses identify devices on the public internet. You can enter the public IP address of an Instance into any browser connected to the Internet, and access content being served from that Instance. 
+
+Depending on the type of resource and its configuration, it may or may not have a public IP address. For example, you can choose to give an [Instance](/compute/instances/) one or more public IP addresses ([IPv4] and/or IPv6), or alternatively, no public IP address at all.
```

Comment:
```suggestion
Depending on the type of resource and its configuration, it may or may not have a public IP address. For example, you can choose to give an [Instance](/compute/instances/) one or more public IP addresses ([IPv4](#ipv4) and/or IPv6), or alternatively, no public IP address at all.
```